julia_src/http.jl

```diff
@@ -1,82 +1,52 @@
 using HTTP, JSON, JuMP
-import Xpress
-include("REopt.jl")
-using GhpGhx
-import REopt as reoptjl
+using HiGHS, Cbc, SCIP
 using GhpGhx
+import REopt as reoptjl  # For REopt.jl, needed because we still have local REopt.jl module for V1/V2
 using DotEnv
 DotEnv.config()
 
 const test_nrel_developer_api_key = ENV["NREL_DEVELOPER_API_KEY"]
 
-function job(req::HTTP.Request)
-    d = JSON.parse(String(req.body))
-    timeout = pop!(d, "timeout_seconds")
-    tol = pop!(d, "tolerance")
-    m = REopt.xpress_model(timeout, tol)
-    @info "Starting REopt with timeout of $(timeout) seconds..."
-	error_response = Dict()
-	results = Dict()
-	try
-    	results = REopt.reopt(m, d)
-	catch e
-		@error "Something went wrong in the Julia code!" exception=(e, catch_backtrace())
-		error_response["error"] = sprint(showerror, e)
-	end
-    optimizer = backend(m)
-	finalize(optimizer)
-    Xpress.postsolve(optimizer.inner)
-	empty!(m)
-	GC.gc()
-	if isempty(error_response)
-    	@info "REopt model solved with status $(results["status"])."
-    	return HTTP.Response(200, JSON.json(results))
-	else
-		@info "An error occured in the Julia code."
-		return HTTP.Response(500, JSON.json(error_response))
-	end
+include("os_solvers.jl")
+
+# Load Xpress only if it is installed, as indicated by ENV["XPRESS_INSTALLED"]="True"
+xpress_installed = get(ENV, "XPRESS_INSTALLED", "False")
+if xpress_installed == "True"
+    import Xpress  # Why not "using Xpress"?
```

Comment:
Why not? :)

Reply:
Updated to "using Xpress", and tested with V2 and V3